scan-command/src/test/java/io/ballerina/scan/utils/RuleDescription.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.scan.utils;
+
+/**
+ * {@code RuleDescription} represents the descriptions for static code analysis rules.
+ *
+ * @since 0.1.0
+ */
+public class RuleDescription {
```

Comment:
Why do we need a separate class? Define the constant in the relevant test file?

Reply:
Imo, It is better to use a separate class, because these constants are use in several test functions.
ex:- CoreRuleTest file and Rule<id> test file

Reply:
Used most in one (where we can have the constant) and just once or twice in the other, right? Since these don't change often either, I think we can avoid the class.